projects/js-packages/connection/components/manage-connection-dialog/stories/index.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ManageConnectionDialog from '..';
+
+export default {
+	title: 'JS Packages/Connection/Manage Connection Dialog',
+	component: ManageConnectionDialog,
+};
+
+const Template = args => <ManageConnectionDialog { ...args } />;
+
+export const _default = Template.bind( {} );
+_default.args = {
+	isOpen: true,
```

Comment:
You could set the default value for the props. `title` in this case.

```suggestion
	isOpen: true,
	title: 'Manage your Jetpack connection',
```

<img width="748" alt="Screen Shot 2022-10-06 at 12 26 05" src="https://user-images.githubusercontent.com/77539/194354167-570807dc-25ff-4eff-af09-3b29094c3066.png">